content/technical/open-source-as-a-service.md

```diff
@@ -0,0 +1,153 @@
+Title: Your Service is not Open Source
+Date: 2020-02-07 18:35
+Tags: open source, technical
+Slug: open-source-as-a-service
+Category: Technical
+Summary: Sharing the source code to your SaaS is insufficient. We effectively enable users to contribute to the a running service for it to be Open Source.
```

Comment:
We effectively **need to** enable users to contribute to ~~the~~ a running service for it to be Open Source.

Reply:
Indeed. Thanks for catching that. I replaced this @gsaslis text.